src/encoded/schemas/changelogs/experiment.md

```diff
@@ -1,5 +1,8 @@
 ## Changelog for experiment.json
 
+### Minor changes since schema version 37
```

Comment:
Since Experiment is still on version 36, I think this entry can be moved below "Minor changes since schema version 36" (after "added 'degron'...)

Reply:
Ah thanks for catching that :) I missed it totally